paper/manuscript.tex

```diff
@@ -48,132 +51,165 @@
+The source code for \refnx\ is held in a version controlled git repository hosted on GitHub.\footnote{https://www.github.com/refnx/refnx} The standard GitHub workflow is followed in which contributors create their own `fork' of the main \refnx\ repository, and create a feature branch to which they make modifications. They then submit a pull request (PR) against the main repository. The modifications made in the PR are checked on continuous integration (CI) web-services that run the test suite against a matrix of Python versions on the macOS, Linux and Windows operating systems. Features are merged into the main repository if all tests pass, and if manual code review concludes that the changes are scientifically correct, of sufficiently high standard, and useful. When a sufficient number of features have accumulated, a new release is made. Successive releases have an incrementing semantic version number which can be obtained from the installed package, with each release being given its own DOI.
+
+The recommended way of using \refnx\ is from a \conda\ environment, which offers package, dependency and environment management \cite{conda}, using the pre-compiled distributables on the \refnx\ conda-forge channel. These distributables are made as part of the release process using the same CI web-services used to test the code. The matrix of distributables covers the major Python versions currently in use, across the macOS, Windows, and Linux operating systems. Alternatively the package can be installed from source, either directly from the git repository, or via \pip\ from the version uploaded to PyPI.\footnote{https://pypi.python.org/pypi/refnx} Building from source requires a C compiler and the \Cython\ and \NumPy\ packages to be installed; further dependencies should be installed to run the test suite to verify that compilation and installation was successful.
+
+% unless refnx has an absolute promise of permanent backwards compatibility, conda packages of the current version will not run old analysis code and so conda won't be usable for the reproducibility aspect.
+
+% with wheels for refnx, is conda any easier than 'pip install refnx'
+
+% FIXME: is only numpy required for building? more is required to run the test suite and people should do so?
```

Comment:
numpy, cython, C compiler for building
The other dependencies are pulled in during install

Reply:
setup.py isn't going to fetch things though is it? (`pip install .` can), so some package manager needs to do that. Just thinking that we shouldn't underspecify the task